task/02-numbers-tasks.js

```diff
@@ -54,7 +52,11 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-    throw new Error('Not implemented');
+  if (value1 + value2 === 0) return 0;
```

Comment:
Do we actually need this `if` statement? `(value1 + value2) / 2` will return the same result